instance/models/server.py

```diff
@@ -23,21 +23,19 @@
 # Imports #####################################################################
 
 import logging
-import novaclient
 import time
 
-from swampdragon.pubsub_providers.data_publisher import publish_data
-
+import novaclient
```

Comment:
@antoviaque Does this come before the `django` and `swampdragon` imports to avoid mixing `import ...` and `from ... import ...` patterns? Looking at the [file](https://github.com/open-craft/opencraft/blob/python3.5/instance/models/server.py) as a whole, the imports section does look nice :) But the ordering is inconsistent with the ordering in other files, where you seemed to be pretty strict about alphabetical ordering. That's why I'm pointing it out :)